encrypt/encrypt_test.go

```diff
@@ -0,0 +1,70 @@
+// Copyright © 2019 Intel Corporation
+//
+// SPDX-License-Identifier: GPL-3.0-only
+
+package encrypt
+
+import (
+	"testing"
+
+	"github.com/GehirnInc/crypt"
+	// package requires import the hash method to blank
```

Comment:
Typo in comment. "import" ==> "importing"

Reply:
done